server/conn.go

```diff
@@ -2167,7 +2179,7 @@ func (cc *clientConn) writeResultset(ctx context.Context, rs ResultSet, binary b
 	return false, cc.flush(ctx)
 }
 
-func (cc *clientConn) writeColumnInfo(columns []*ColumnInfo, serverStatus uint16) error {
+func (cc *clientConn) writeColumnInfo(ctx context.Context, columns []*ColumnInfo, serverStatus uint16) error {
```

Comment:
ctx is not used?

Reply:
Fixed this in 04c8d575e17b3f3c253eac7f4d2a094ddd88cb0a